src/reporting/quicksight-custom-resource.ts

```diff
@@ -111,16 +111,6 @@ export function createQuicksightCustomResource(
             columnGeographicRoles: ['COUNTRY'],
           },
         ],
-        datasetParameters: [
-          {
-            DateTimeDatasetParameter: {
-              Id: 'a05a5ad5-c6b1-40de-9f96-f2bfcc5d2c21',
-              Name: 'EventStartDate',
```

Comment:
Did we move this criteria to SQL itself?

Reply:
"datasetParameters" is a configuration used for adding parameters to the QuickSight dataset, other than SQL, is unnecessary and can be safely removed.